models/issue_reaction.go

```diff
@@ -18,7 +18,7 @@ import (
 // Reaction represents a reactions on issues and comments.
 type Reaction struct {
 	ID          int64          `xorm:"pk autoincr"`
-	Type        string         `xorm:"INDEX UNIQUE(s) NOT NULL"`
+	Type        string         `xorm:"VARCHAR(191) INDEX UNIQUE(s) NOT NULL"`
```

Comment:
why is a limit being added here ?

Reply:
@strk please take a look at https://github.com/go-gitea/gitea/pull/3516#issuecomment-366648794 where I explained where that limit comes from. In short, it's for compatibility with MariaDB/MySQL versions that run the InnoDB 5.6 engine as we cannot have indexed fields that are > 767 bytes long (191x 4 bytes for utf8mb4).